src/components/wallet/TopCards/AddressCard/ChainSelect.vue

```diff
@@ -1,14 +1,22 @@
 <template>
     <div class="chain_select">
         <button @click="setChain('X')" :active="chain === 'X'">X</button>
-        <button @click="setChain('P')" :active="chain === 'P'">P</button>
-        <button @click="setChain('C')" :active="chain === 'C'" v-if="isEVMSupported">C</button>
+        <button @click="setChain('P')" :active="chain === 'P'" v-if="walletType !== 'multisig'">
```

Comment:
i think P button should be activated and disable the X chain

Reply:
@rafaelkordonias and please fix the styling
![Screenshot 2023-05-24 at 13 05 01](https://github.com/chain4travel/camino-wallet/assets/58547974/dfb007d2-8848-495c-b43b-3499747b40ae)